src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/UpToDate/BuildUpToDateCheck.cs

```diff
@@ -475,6 +476,8 @@ private bool CheckCopyToOutputDirectoryFiles(BuildUpToDateCheckLogger logger, ID
             var items = _items.SelectMany(kvp => kvp.Value).Where(item => item.CopyType == CopyToOutputDirectoryType.CopyIfNewer)
                 .Select(item => item.Path);
 
+            string outputFullPath = Path.Combine(_msBuildProjectDirectory, _outputPath);
```

Comment:
Rename `_outputPath` to make it clear that it is a relative path instead of a full path?

Reply:
It actually could be both, so I've named it as such.